readthedocs/core/middleware.py

```diff
@@ -139,7 +139,7 @@ def process_response(self, request, response):
                         )
 
                         # NOTE: This was added to support the fallback cookie
-                        if not settings.SESSION_COOKIE_SAMESITE:
+                        if settings.SESSION_COOKIE_SAMESITE != "None":
```

Comment:
Does this works when this setting is set to `False`? It seems it's a valid value: https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-SESSION_COOKIE_SAMESITE

Reply:
We need to send `None`, False will use the browsers default. None will send the cookie on cross-site requests, we need that.